reports: Section's options not fully shown fixed (#5172)
zaproxy/zap-extensions: addOns/reports/CHANGELOG.md

## Unreleased

> 
> 
> ### Fixed

Reviewer: ```suggestion
### Fixed
```

> - Fixed Sections Option not fully shown in Generate Report Dialog.

Reviewer: ```suggestion
- Ensure the Sections' options are fully shown always in Generate Report Dialog (Issue 8259).
```

Reviewer: Didn't apply waiting for other opinions about the wording.

## [0.27.0] - 2023-12-19

> 

Reviewer: ```suggestion
```
